rdkcentral/rdkservices
# DELIA-43653 : Add Dolby MS12 APIs to displaysettings plugin

Reason for change: 1) Added dolby APIs mentioned in the ticket
2) Added Volume gain and level adjustment APIs
3) Modified existing dolby APIs to take audioPort arg,
default is HDMI0
Test Procedure: Verify using curl commands
Risks: None

Signed-off-by: Deekshit Devadas <deekshit.devadasy@sky.uk>

RDK-29271 : DisplaySettings Thunder plugin changes

Reason for change: Added Mute and volume control
APIs
Test Procedure: Verify volume control on amlogic
platform using curl commands
Risks: None

Signed-off-by: Deekshit Devadas <deekshit.devadasy@sky.uk>
